docs/accounts-and-projects/projects.md

```diff
@@ -44,32 +52,31 @@ The project will also need to be added to the OSG topology at
 
 https://github.com/opensciencegrid/topology/tree/master/projects
 
-1. Clone the topology repository and `cd` into it.
-```
-git clone https://github.com/opensciencegrid/topology
-cd topology
-```
+1.  Clone the topology repository and `cd` into it.
 
-2. Run the "next project id" script in the bin/ directory:
-```
-bin/next_project_id
-```
-That will give you the ID number for the new project.  
+        git clone https://github.com/opensciencegrid/topology
+        cd topology
+
+2.  To add a new project file, you will need to ensure you are working
+    within your own (forked) version of the repository. If you have not already
+    done so, fork the topology repository by visiting https://github.com/opensciencegrid/topology, 
+    clicking "Fork". Detailed information for setting up and maintaining your fork can be found
+    in the [GitHub documentation page](../documentation/github.md).
+
+    Within your forked repository, go into the projects folder and create a project file. 
+    The project file should be named the same as the short (UNIX) project name. 
+    It usually makes sense to copy from another project 
+    file that was also created for an OSG Connect project so the 
+    sponsor information is already correct.  Fill in the appropriate information.
```

Comment:
Instead of this, maybe we should tell them to go here https://github.com/opensciencegrid/topology/new/master/projects and copy in the contents of https://github.com/opensciencegrid/topology/blob/master/template-project.yaml then providing instructions for proper naming of the file, modifying attrs, etc. Then they can create a new branch and submit a PR